packages/calypso-products/src/plans-list.tsx

```diff
@@ -3609,3 +3619,20 @@ if ( isEnabled( 'plans/migration-trial' ) ) {
 		getTitle: () => i18n.translate( 'Business Trial' ),
 	};
 }
+
+if ( isEnabled( 'plans/hosting-trial' ) ) {
```

Comment:
I don't feel like we need the feature flag here. It seems ok for Calypso to have support for the new constant even in production. All we want to guard against is the user seeing the UI that allows them to select the plan for now.